### DISCUSSION TITLE
Oxalis-AS4 5.4.0 Release

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/Oxalis-AS4#191] [General]
- Bump oxalis to version 5.4.0 and used Oxalis Community version of peppol specification : 4caecd7a5ee17e493507f50a819443edc8615464
- Excluded woodstox-core dependency: 930c78df9167998add8f6257ac05700dbdfcae09

<hr /><em>This discussion was created from the release <a href='https://github.com/OxalisCommunity/Oxalis-AS4/releases/tag/v5.4.0'>Oxalis-AS4 5.4.0 Release</a>.</em>

**Comment by Kim-Billit** [comment]
Will the dist zipfiles also be made available?
Thx!

**Comment by aaron-kumar** [comment]
Yes, @Kim-Billit  please check release now: https://github.com/OxalisCommunity/Oxalis-AS4/releases/tag/v5.4.0 
Though you can always find those in maven repo at specified artifact location e.g. https://repo1.maven.org/maven2/network/oxalis/oxalis-as4/5.4.0/

**Comment by Kim-Billit** [comment]
Thx!
